panels/UserPanel.php

```diff
@@ -116,4 +108,36 @@ protected function dataToString($data)
 
         return VarDumper::export($data);
     }
+
+    /**
+     * Returns an array containing information about the logged-in user.
+     *
+     * The array should contain two items:
+     * - the model that should be set on [[\yii\widgets\DetailView::model]]
+     * - the array that should be set on [[\yii\widgets\DetailView::attributes]]
+     *
+     * @param IdentityInterface $identity
+     * @return array
+     */
+    protected function identityData(IdentityInterface $identity)
+    {
+        if ($identity instanceof Model) {
+            $data = $identity->getAttributes();
+            $attributes = [];
+
+            foreach ($data as $attribute => &$value) {
```

Comment:
@dynasource Ah interesting! You think I should either remove the reference or switch to looping over `array_keys()` ?